loadgenerator/loadgen

```diff
@@ -400,6 +399,45 @@ def _check_prereqs():
     except subprocess.CalledProcessError:
         return False
 
+#####################################
+# CLOUD RUN LOAD GENERATOR COMMANDS #
+#####################################
+
+def run_auto(args):
+    """Create automatic load generator and Web UI for normal traffic pattern.
+    Use Cloud Run to host the container and serve the Locust Web UI.
+    args.target_address -- the IP address to send load (frontend)
+    """
+    # _setup_cloud_run()
+    name = _generate_job_name()
+    logger.info('Creating load generation job {0}...'.format(name))
+
+    if not _is_address_accessible(args.target_address):
+        logger.error('Could not access target address {0}'.format(args.target_address))
+        print('Please double check the target address and try again.')
+        return
+
+    command = (
+        'gcloud run deploy {0}'
+        ' --image gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator:gz'
```

Comment:
the tag should be the release tag or alternatively, if we push the loadgen image to the project where sandbox is created, it should be latest tag from that project.

Reply:
Yeah I was going to replace it with latest. Hopefully this helps with testing (I didn't push a new latest tag, used a gz tag). I'll also replace the GCE image with the one tagged by commit so it doesn't change.